util/proxy/transport.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package proxy
+
+import (
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// NewTransport creates a Kubernetes API Server proxied, port-forwarded
+// HTTP(S) transport
+func NewTransport(s Scope) (*http.Transport, error) {
```

Comment:
Where is this function used/going to be used?

Reply:
Was used for testing. Now deleted.